README.md

```diff
@@ -279,17 +279,6 @@ attempt. The [`disconnect`] function suspends until the peripheral has settled o
 peripheral.disconnect()
 ```
 
-_If the underlying subsystem fails to deliver the disconnected state then the [`disconnect`] call could potentially
-stall indefinitely. To prevent this (and ensure underlying resources are cleaned up in a timely manner) it is
-recommended that [`disconnect`] be wrapped with a timeout, for example:_
-
-```kotlin
-// Allow 5 seconds for graceful disconnect before forcefully closing `Peripheral`.
-withTimeoutOrNull(5_000L) {
-    peripheral.disconnect()
-}
-```
-
```

Comment:
This was an extra precaution that I don't think was needed.
In all my testing, the Android system always properly calls the callback when a device disconnects.